profiles/dictionary/common.yaml

```
@@ -16,13 +16,13 @@ name:
 id:
   title: ID
   description: A property reserved for globally unique identifiers. Examples of
-    identifiers that are unique include UUIDs and DOIs.
+    identifiers that are unique include UUIDs and DOI links.
```

```suggestion
    identifiers that are unique include UUIDs and DOIs.
```

Reverting this one, it is fine to just call it DOIs here.